platform_umbrella/apps/verify/test/redis_test.exs

```diff
@@ -0,0 +1,90 @@
+defmodule Verify.RedisTest do
+  use Verify.TestCase, async: false
+
+  @moduletag :cluster_test
+
+  @new_path "/redis/new"
+  @new_redis_header Query.css("h3", text: "New Redis Instance")
+  @name_field Query.text_field("redis_instance[name]")
+  @save_button Query.button("Save Redis")
+  @size_select Query.select("Size")
+  @type_select Query.select("Type")
+
+  defp show_page_header(instance_name), do: Query.css("h3", text: instance_name)
+
+  test "can start a standalone redis instance", %{session: session} do
+    instance_name = "int-test-#{:rand.uniform(10_000)}"
+
+    session
+    # create new instance
+    |> visit(@new_path)
+    |> assert_has(@new_redis_header)
+    |> fill_in(@name_field, with: instance_name)
+    |> click(@save_button)
+    # verify show page
+    |> assert_has(show_page_header(instance_name))
+    # Assert that the first pod for the cluster is shown
+    |> assert_has(Query.css("tr:first-child", text: "#{instance_name}-0"))
+
+    # Assert that we have gotten to the show page
+    path = current_path(session)
+    assert path =~ ~r/\/redis\/[\d\w-]+\/show$/
```

Comment:
We'll need some way to verify pods are started. That along with battery install will likely common patterns that we'll need

Reply:
Yeah, are you thinking that we'd use the k8s client to do that? Or go to the pod show page and check the status?